ext/civicrm_search_ui/ang/afsearchBadgedLayoutForm.aff.php

```diff
@@ -0,0 +1,11 @@
+<?php
+use CRM_CivicrmSearchUi_ExtensionUtil as E;
+
+return [
+  'type' => 'search',
+  'title' => E::ts('Badged Layout Form'),
+  'description' => E::ts('Badged Layout Form'),
+  'icon' => 'fa-list-alt',
+  'server_route' => 'civicrm/searchkit_ui/admin/badgelayout',
+  'modified_date' => '2023-12-05 09:31:54',
```

Comment:
```suggestion
  'server_route' => 'civicrm/searchkit_ui/admin/badgelayout',
```

Reply:
Modified date doesn't belong in a packaged form. I've also opened https://github.com/totten/civix/pull/322 to fix this upstream.